### PR TITLE
PR: Add QtHelp Wrapper

### LINKED ISSUE #127: Add Wrapper for QtHelp [feature request]
no problem importing QtHelp from PyQt5, but cannot import from qtpy.

No problem:

`from PyQt5 import QtHelp`

Module not found:

`from qtpy import QtHelp`

### PR DESCRIPTION
**Pull request by rlaverde**
Fixes #127

### CODE REVIEW

File: qtpy/QtHelp.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from . import PYQT4
+from . import PYSIDE
+from . import PYSIDE2
+
+if PYQT5:
+    from PyQt5.QtHelp import *
+elif PYSIDE2:
+    # Current wheels don't have this module
+    # from PySide2.QtHelp
+    pass
```
Comment (goanpeca) [inline qtpy/QtHelp.py]: But, this would result in a successful import. Should we at least add a warning?

@ccordoba12 ?

Comment (rlaverde) [inline qtpy/QtHelp.py]: If we are going to add a warning we should add it to qtmultimedia too

Comment (ccordoba12) [inline qtpy/QtHelp.py]: Yeah, we should add warning to this one and also QtMultimedia.

**Comment by rlaverde** [comment]
I created a custom warning? is that ok? or should I just use the standard warning ` warnings.warn()` as #121

**Comment by ccordoba12** [comment]
> I created a custom warning? is that ok?

I think it's ok, thanks!

**Pull request merged**